### ISSUE TITLE
Ability to organize Sales Reps by First Name (rather than by Last Name)

### ISSUE THREAD

**Issue by derekantrican** [derekantrican/SalesMap#5]
E.G.

```
Frodo Baggins
Dorothy Gale
Buzz Lightyear
Harry Potter
```

would become

```
Buzz Lightyear
Dorothy Gale
Frodo Baggins
Harry Potter
```

**Comment by derekantrican** [comment]
Relevant:
https://msdn.microsoft.com/en-us/library/b0zbh7b6(v=vs.110).aspx
http://stackoverflow.com/questions/15275269/sort-a-list-from-another-list-ids

**Comment by derekantrican** [comment]
Likely solution: http://stackoverflow.com/questions/3309188/how-to-sort-a-listt-by-a-property-in-the-object

**Comment by derekantrican** [comment]
This will be implemented in v6.0
